extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -117,6 +124,24 @@ export class Bind {
      */
     this.digestQueuedAfterScan_ = false;
 
+    /**
+     * @const @private {!Array<Element>}
+     */
+    this.dynamicRoots_ = [];
```

Comment:
In order to prevent nodes that aren't the roots of dynamic elements from triggering a remove and an add. When I was testing with this, the newly added element would also trigger the mutation observer  with itselrf as the mutation target.